### PR TITLE
Small updates to the README file + screenshots.

### PR DESCRIPTION
**Pull request by mithro**
Signed-off-by: Tim 'mithro' Ansell <me@mith.ro>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
-# Sphinx Symbiflow Theme
+# SymbiFlow's Sphinx Theme (Based on Material Design).
+
+This repository contains the [Sphinx](https://www.sphinx-doc.org/en/master/)
+ theme used
+[by SymbiFlow](https://symbiflow.github.io)
+on all it's generated documentation
```
Comment (daniellimws) [inline README.md]: `its`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
-# Sphinx Symbiflow Theme
+# SymbiFlow's Sphinx Theme (Based on Material Design).
+
+This repository contains the [Sphinx](https://www.sphinx-doc.org/en/master/)
+ theme used
+[by SymbiFlow](https://symbiflow.github.io)
+on all it's generated documentation
+([including the theme itself!](https://github.com/SymbiFlow/sphinx_symbiflow_theme)).
```
Comment (daniellimws) [inline README.md]: * Wanna link to the generated documentation (https://sphinx-symbiflow-theme.readthedocs.io/en/latest/) instead?
* including the theme's itself

**Review by daniellimws — CHANGES_REQUESTED** [review]

**Comment by daniellimws** [comment]
Some minor changes on the phrasing

File: README.md (hunk 1 above)
Comment (mithro) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (mithro) [reply README.md]: Change the link.

Change the wording to `include the theme's own!`.

**Review by daniellimws — APPROVED** [review]

**Pull request merged**